client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -65,28 +65,22 @@
 
 .site-title__input-hint {
 	display: flex;
-	color: var( --studio-gray-20 );
+	color: var( --studio-gray-50 );
 	font-family: $default-font;
-	font-size: $default-font-size;
+	font-size: 14px;
```

Comment:
Odd, how come the variable didn't give us right size already?

See https://github.com/Automattic/wp-calypso/blob/c06383b8b1ad35b8741f1f898d5385ec0d97bf20/client/assets/stylesheets/shared/_typography.scss#L48-L49

Reply:
Thanks for pointing this out @simison

It's because we weren't using the correct variable.
https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_variables.scss#L8 is an absolute value and we transitioned to using relative ones in Calypso.

We should replace the font-size declaration with `@include onboarding-medium-text` which is setting the font-size to `$font-body-small` and as best practice we should avoid setting absolute font-sizes.

cc: @yansern